css-view-transitions-2/Overview.bs

```diff
@@ -983,6 +1080,56 @@ When capturing the old or new state for an element, perform the following steps
 	Note: This is written in a monkey-patch manner, and will be merged into the algorithm once the L1 spec graduates.
 </div>
 
+## Capturing and applying 'view-transition-group' ## {#vt-group-algorithm}
+
+<div algorithm="additional old apture steps (group)">
+When capturing the old or new state for an element, perform the following steps given a [=captured element=] |capturedElement|, an [=element=] |element| and a {{ViewTransition}} |transition|:
+
+	1. Let |containingGroupName| be the [=used value=] of |element|'s ''view-transition-group''.
+
+	1. If |containingGroupName| is not null, then:
+
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][containingGroupName].
+
+		1. If |groupContainerElement| is not null, then:
+
+			1. Let |containerElement| be the old element that is being captured for |containingGroupName|.
+
+			1. If |containerElement| is an ancestor of |element|, then:
+
+				1. Set |capturedElement|'s [=captured element/old transform=] to the transform that would map |element|’s [=border box=] from | groupContainerElement|'s [=border box=] to its current visual position.
+
+				1. Set |capturedElement|'s [=captured element/containing group=] to |containingGroupName|.
+</div>
+
+<div algorithm="additional new capture steps (group)">
+When capturing the new state for an element, perform the following steps given a [=captured element=] |capturedElement| and an [=element=] |element|:
+
+	1. Set |capturedElement|'s [=captured element/containing group=] to the [=used value=] of |element|'s 'view-transition-group'.
```

Comment:
Can you link to the issue for deciding the hierarchy mismatch situation. That's TBD.

Note for the current spec text: it's overriding the parent from the old DOM, if the name existed in the old DOM, without updating the old transform which seems sketchy. On the flip side, if we keep the ancestor from the old DOM but it doesn't exist in the new DOM then how do we compute the new transform..? Reviewing this, I'm more convinced of the use the LCA option.

Reply:
Done